### ISSUE TITLE
Last.fm Auth redirects to another Application.

### ISSUE THREAD

**Issue by srcnalt** [AurousGroup/frontend#63]
I was trying to work on last.fm scrobbler for Aurous with the api key that is used for fetching song infos in the js files. But when I tried to get a permission for that key It was requesting permission for an app called "XSPF Jukebox".

**Comment by miracle091** [comment]
Maybe is this? https://github.com/lacymorrow/xspf-jukebox

**Comment by srcnalt** [comment]
Exactly this one, there sould be a spesific account for Aurous to get its own api key.

**Comment by Codeusa** [comment]
Since LAST.FM refuses to actually let anyone register api accounts right now, we just used another apps.

**Comment by srcnalt** [comment]
That's correct, api register page leads you back to the home page. I'll ask them if this feature will be back soon.
